components/UI/Map/VectorMap/OknSource.tsx

```diff
@@ -1,52 +1,91 @@
-import React, { useEffect } from 'react';
 import { Source, Layer, useMap } from 'react-map-gl';
-import type { CircleLayer, FillLayer, LineLayer } from 'react-map-gl';
 import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import type { CircleLayer, FillLayer, LineLayer } from 'react-map-gl';
 import { activeFilterSelector, activeFilterParamsSelector } from 'state/features/selectors';
-import { FilterType } from 'components/UI/Filters/Filters.types';
 import { AREA_CONFIG, OBJECTS_CONFIG } from 'components/Model/OKN/Okn.constants';
+import { FilterType } from 'components/UI/Filters/Filters.types';
+import { getLayerActiveStyle } from 'components/helpers/activeObject';
 import { MapItemType } from 'common/types/map-item';
 import { OknAreaType } from 'common/data/okn/oknConstants';
 import { usePopup } from '../providers/usePopup';
+import useMapHoverObject from '../providers/useMapHoverObject';
 
-const OKN_LAYER_ID = 'ekb-okn-layer';
+const LAYERS = {
+    points: {
+        data: '/ekb-okn.json',
+        layer: 'ekb-okn-layer',
+        source: 'ekb-okn-source',
+        zone: null,
+    },
+    protect: {
+        data: '/ekb-okn-protect.json',
+        layer: 'ekb-okn-protect-layer',
+        source: 'ekb-okn-protect-source',
+        zone: OknAreaType.ProtectZone,
+    },
+    security: {
+        data: '/ekb-okn-security.json',
+        layer: 'ekb-okn-security-layer',
+        source: 'ekb-okn-security-source',
+        zone: OknAreaType.SecurityZone,
+    },
+    objects: {
+        data: '/ekb-okn-objects.json',
+        layer: 'ekb-okn-objects-layer',
```

Comment:
lets rename to layerId, sourceIds, dataPath?)

Reply:
Done